src/+replab/+infra/resolveRef.m

```diff
@@ -119,9 +119,9 @@
     
     % Circumventing direct assignment for octave
     if length(parts) == 1
-        newParts = '';
+        newParts = {};
     else
-        newParts = parts{2:end};
+        newParts = parts(2:end);
     end
-    el = headElement.get(newParts);
```

Comment:
@jdbancal  `newParts` needs to be a cell array; thus we index it with `()`, otherwise, it returns multiple values, and we grab only the first one -- then the method list on classes fail for example in the reference help.